.github/ISSUE_TEMPLATE/FEATURE_REQUEST.yml

```diff
@@ -0,0 +1,68 @@
+name: ➕ Feature Request
+description: File a file request if you have a suggestion for a new feature.
+title: "[FR]: "
+labels: []
+body:
+  - type: markdown
+    attributes:
+      value: |
+        For large or ambiguous requests, such as significant breaking changes
+        or use cases that could be addressed by multiple different features,
+        consider instead starting a [Pitch][1] discussion to discuss with
+        the Firebase community.
+
+        A good feature request ideally
+        * is either immediately obvious (e.g. _Add Sign in with Apple support_), or
+        * starts with a use case that is not achievable with the existing
+          Firebase API and includes a brief API proposal that would make the use
+          case possible.
+
+        <!--- Links -->
+        [1]: https://github.com/firebase/firebase-ios-sdk/discussions/categories/pitches
+  - type: textarea
+    attributes:
+      label: Description
+      description: |
+        Describe the feature request. Explain the following:
```

Comment:
wording nit: 
    
    Describe the feature request. Provide the following:
        1. Use case
        2. How the Firebase SDK _currently_ handles it
        3. How you propose the Firebase SDK _should_ handle it

Reply:
Done in [5163007](https://github.com/firebase/firebase-ios-sdk/pull/10120/commits/516300786873a1c8f614911a329c4156618918dd)